src/services/journal.js

```diff
@@ -0,0 +1,97 @@
+import createResponse from '../utils/createResponse';
+export default class Journal {
+
+    constructor(redis) {
```

Comment:
All these logs are related to Redis, you can use `context.logGroupName` to group them together, so that in Cloudwatch they will be in their own section, rather than polluting the main Cloudwatch log